Example-Crypto-REXX/README.md

```diff
@@ -0,0 +1,110 @@
+## Example-Crypto-REXX
+
+This sample retrieves crypto information from image activation profiles associated with LPARs located on a specific CPC. It stores the results, which are in .csv format, in a z/OS data set member.  You can import the content into Microsoft™ Excel or another application of your choice.
+
+This sample uses **HWIREST API** to do the following:
+- List CPCs and retrieve the URI and target name associated with the
+ LOCAL CPC (default) or the *CPCname* provided
+- List the LPARs on the designated CPC and query the LPAR status.
```

Comment:
minor nit, but the lpar status comes back with the list response, so you're not actually querying it, would you change this to:
'List the LPARs on the designated CPC.'